plugins/output/flight/flight.go

```diff
@@ -47,27 +48,86 @@ func (f *Flight) Write(metrics []telegraf.Metric) error {
 	builder := array.NewRecordBuilder(memory.DefaultAllocator, &f.schema)
```

Comment:
The "Write()" function could use some more comments in general.

Reply:
Is this okay or do you think I should do more comments?